apps/src/templates/sectionProgressV2/progress-table-v2.module.scss

```diff
@@ -214,10 +214,14 @@ $level-cell-width: 52;
       justify-content: space-evenly;
       overflow-x: hidden;
       flex-wrap: nowrap;
-    }
 
-    .expandedHeaderLevelCellUnexpanded {
-      background-color: white;
+      &LevelNumber {
+        margin-top: 0.0625em;
```

Comment:
If we end up using margin, I'm wondering if we can move the star up just a wee bit more, such that the bottom of the star lines up with the bottom of the text. Because the star is bottom heavy, at normal zoom levels it still looks a tiny bit lower in alignment right now compared to the text.
![image](https://github.com/code-dot-org/code-dot-org/assets/2933346/5eea537c-f337-4056-9b16-2e15721d6ac7)

Reply:
From office hours - probably a font issue with Metropolis. Comparing between Firefox, Chrome, and Edge on Windows at varying browser zoom levels and screen zoom levels, Eric's margin seemed to have the least variance, so we will keep it as the PR is currently written for now.